tests/tas-mtls/src/test/java/org/alfresco/rest/MtlsRestTest.java

```diff
@@ -26,34 +34,29 @@
 @ContextConfiguration ("classpath:alfresco-mtls-context.xml")
 public abstract class MtlsRestTest extends AbstractTestNGSpringContextTests
 {
-    private static Logger LOGGER = LogFactory.getLogger();
+    protected static Logger LOGGER = LogFactory.getLogger();
 
     @Autowired
-    protected ServerHealth serverHealth;
-
-    @Autowired
-    protected RestWrapper restClient;
-
+    protected TestMtlsProperties testMtlsProperties;
     @Autowired
     protected DataUserAIS dataUser;
-
     @Autowired
-    protected DataSite dataSite;
-
-    @Autowired
-    protected DataContent dataContent;
-
-    protected SiteModel testSiteModel;
+    protected RestWrapper restClient;
 
-    CloseableHttpClient client = HttpClients.createMinimal();
+    private CloseableHttpClient client = HttpClients.createMinimal();
 
     @BeforeSuite (alwaysRun = true)
-    public void checkServerHealth() throws Exception
+    public void setupSSLConfig() throws Exception
     {
         super.springTestContextPrepareTestInstance();
-        serverHealth.isServerReachable();
-        serverHealth.assertServerIsOnline();
-        testSiteModel = dataSite.createPublicRandomSite();
```

Comment:
No feasible way of setting SSL context, I've switched to relying solely on RestAssured (that's used in restClient).